### ISSUE TITLE
Ctrl+Shift+S should be the shortcut for Save All

### ISSUE THREAD

**Issue by rkeithhill** [Microsoft/vscode#1760]
I think this is another case where VSCode should follow the path paved by its namesake - Visual Studio.  Ctrl+Shift+S is `Save All Files` in VS and I think it should be the same in VSCode.  I use "save all files" **way** more often than saving a file to a different name.  That is actually a pretty rare occurrence for me.  And VSCode already handles the case where you press Ctrl+S to save a file that doesn't have a name yet.  That particular case is by far the most common case for "Save as" and it's already covered by Ctrl+S.  Plus it helps the transfer of learning when coming over from Visual Studio.

**Comment by Tyriar** [comment]
I've seen both and prefer <kbd>ctrl</kbd>+<kbd>shift</kbd>+<kbd>s</kbd> to save as personally.

You should be able to change this locally by opening your keybindings file (<kbd>ctrl</kbd>+<kbd>shift</kbd>+<kbd>p</kbd>, type "keyboard", <kbd>enter</kbd>) and adding this:

```js
{ "key": "ctrl+shift+s", "command": "workbench.action.files.saveAll" },
```

**Comment by rkeithhill** [comment]
Just out of curiosity, how often do you actually save a file to a different filename other than the initial save of an untitled file?  I do this exceedingly rarely.  

It would be cool if the VSCode team was getting telemetry info on what features, such as shortcuts, were being used and how often.

**Comment by Tyriar** [comment]
Not that often, but when I do I expect it to be there as it's a pretty standard. Atom and Sublime both feature this keybinding.

**Comment by rkeithhill** [comment]
>> it's a pretty standard

I guess that depends on what you think the "standard" is?  Atom/Sublime or Visual Studio.  :-)

**Comment by stkb** [comment]
I agree that a keybinding for Save All is more useful than Save As, and have customized it for myself too. As for the question for what the default should be, the vscode team walk a difficult line between attracting Visual Studio users and Sublime or Atom or other users.

Maybe it should be a separate issue but I've wondered if vscode should have menu items to let a new user set all the applicable keyboard shortcuts to be the same as their previous/other editor of choice.

**Comment by bpasero** [comment]
We might want to release an extension to make VS users happy but that should be an extension and not defined from VS Code itself.

**Comment by egamma** [comment]
@waderyan fyi

**Comment by gianpaj** [comment]
It's great there is a way to set this shortcut.

I would like to share my workflow when developing a hybrid app with Ionic or WordPress app anything connected to LiveReload or similar development environment and the reason I save multiple files at the same time.

Sometimes you have multiple files opened (SASS, JS/TS, HTML) and you have made some changes that will impact the entire app to be reloaded. Since this can take few seconds, saving one file at the time will trigger the watcher to rebuild the entire app multiple times. Making a single Save All command will trigger only the rebuild and reload only once.

**Comment by Tyriar** [comment]
Another option for this now is to Install a keymap for the program you're migrating from:  https://marketplace.visualstudio.com/search?term=keymap&target=VSCode&category=All%20categories&sortBy=Relevance
